src/cpp/session/modules/SessionTutorial.cpp

```diff
@@ -267,10 +245,12 @@ void handleTutorialHomeRequest(const http::Request& request,
                << tutorial.description
```

Comment:
The examples make me wonder if we should interpret Markdown here. We already build and include a self-contained Markdown parser (`sundown`, see R Presentations for use) so might be very straightforward to integrate. What do you think?

Reply:
That is an excellent idea!